pkg/scheduler/framework/util.go

```diff
@@ -262,3 +263,28 @@ func (nl *NodeLister) List() ([]*v1.Node, error) {
 	}
 	return nodes, nil
 }
+
+// PredicateNodes returns the nodes that can run the task.
```

Comment:
Can we just refactor `PredicateNodes` in `pkg/scheduler/util/predicate_helper.go`?

Reply:
Done